include/mata/ord-vector.hh

```diff
@@ -198,6 +198,8 @@ public:
     virtual void insert(const OrdVector& vec) {
         assert(vectorIsSorted());
         assert(vec.vectorIsSorted());
+
+        // TODO: sometimes it is not efficient to copy both vectors to create a new one
```

Comment:
Are you sure the current merge is slow?
I have an idea for an inplace merge, using a trick that swaps elements between the two vectors, but is it worth it?

Reply:
For concatenation it makes just a small sense to me (we are adding items to the back, which seems to be fast now). But I guess there are some operations using this Union, so maybe we could take a look.